server/container/core/src/main/java/org/apache/james/server/core/MimeMessageInputStreamSource.java

```diff
@@ -78,7 +79,9 @@ public MimeMessageInputStreamSource(String key, InputStream in) throws Messaging
         // Create a temp file and channel the input stream into it
         try {
             out = new DeferredFileOutputStream(THRESHOLD, "mimemessage-" + key, ".m64", TMPDIR);
-            IOUtils.copy(in, out);
+            BufferedOutputStream buffered = new BufferedOutputStream(out, THRESHOLD);
+            IOUtils.copy(in, buffered);
+            buffered.flush();
```

Comment:
the buffered does not need to be closed?

Reply:
Could you make that remark directly on the contributor PR?

Reply:
of course, I just wanted to be sure I was not missing something